src/Shouldly/ShouldlyExtensionMethods/ShouldBeDictionaryTestExtensions.cs

```diff
@@ -3,71 +3,83 @@
 using System.Diagnostics;
 using JetBrains.Annotations;
 
-namespace Shouldly 
+namespace Shouldly
 {
     [DebuggerStepThrough]
     [ShouldlyMethods]
-    public static class ShouldBeDictionaryTestExtensions 
+    public static class ShouldBeDictionaryTestExtensions
     {
         public static void ShouldContainKey<TKey, TValue>(this IDictionary<TKey, TValue> dictionary, TKey key)
+            where TKey : notnull
```

Comment:
This is necessary in order to use `TKey` for `IDictionary<TKey, TValue>` due to the generic constraint added to `IDictionary<,>` in the BCL.